docs/reference/settings/security-settings.asciidoc

```diff
@@ -1015,8 +1015,9 @@ the following settings:
 `username_pattern`::
 (<<static-cluster-setting,Static>>)
 The regular expression pattern used to extract the username from the
-certificate DN. The first match group is the used as the username.
-Defaults to `CN=(.*?)(?:,\|$)`.
+certificate DN. The username is used for auditing and logging (not role mapping).
```

Comment:
The `not role mapping` isn't strictly true (and I expect some readers to be confused).

If you use the role mapping API, then you can map on username.

It's also use for delegated authorization (lookup realms) which is the scenario that (I think) prompted this clarification.

Reply:
Thanks Tim ! I missed those usages.  I have updated the docs.